modules/ROOT/pages/B2B-overview.adoc

```diff
@@ -0,0 +1,46 @@
+= B2B Solutions and Overview
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== B2B Solutions and Overview
+
+B2B (Business-to-Business) is the use of API's or Electronic Data Interchange (EDI) by enterprises to carry out business transactions such as purchasing goods from one another. B2B messages are typically sent over transports such as HTTP(s), FTP, FTPS, SFTP, JMS, and exchanges such as AS2. MuleSoft's B2B solutions integrate seamlessly into the Mule runtime.
```

Comment:
When we refer to the Mule runtime, we have to call it Mule runtime engine. However, since I'm not 100% sure that we are enforcing this convention, I'll defer to @Mulesoft-Mysti to confirm

Reply:
It should be Mule runtime engine on first occurrence. If you mention it more than once, you can shorten to Mule. Just don't say Mule runtime. And yes, we are trying to scrub all the docs to match this convention.

Reply:
Here are the fixes:
B2B (Business-to-Business) is the use of APIs or Electronic Data Interchange (EDI) by Enterprises to carry out business transactions such as purchasing goods from one another. B2B messages are typically sent over transports such as HTTP(s), FTP, FTPS, SFTP, JMS, and exchanges such as AS2. MuleSoft’s B2B solutions integrate seamlessly into the Mule runtime engine.

Reply:
why is Enterprises captialized? Everything else looks good.

Reply:
It's not a specific Enterprise, so it should not be capped. Thanks for the review.